kDriveCore/Data/Models/Upload/UploadFile.swift

```diff
@@ -60,6 +60,8 @@ public class UploadFile: Object, UploadFilable {
     @Persisted(primaryKey: true) public var id = ""
     @Persisted public var name = ""
     @Persisted var relativePath = ""
+    @Persisted var fileProviderItemIdentifier: String?  // NSFileProviderItemIdentifier if any
```

Comment:
A migration is needed
```
kDriveCore/Logging.swift:58: Fatal error: Failed creating realm Migration is required due to the following errors:
- Property 'UploadFile.fileProviderItemIdentifier' has been added.
- Property 'UploadFile.assetLocalIdentifier' has been added.
```

Reply:
Interesting, a simple add only of a field needs a migration, unlike `Mail` ?

Reply:
Did the migration, managed to keep track of the PHAsset id during the migration when possible.
Tested a migration with both normal uploaded files and photo sync, all good.